backend/tournesol/views/video.py

```diff
@@ -110,9 +112,10 @@ def get_queryset(self):
                 queryset = queryset.filter(publication_date__gte=date_gte)
             except ValueError:
                 raise ValidationError('"date_gte" is an invalid datetime')
-        language = request.query_params.get('language') \
-            if request.query_params.get('language') else ""
-        queryset = queryset.filter(language=language) if language else queryset
+
+        languages = request.query_params.get('languages').split(",") \
+            if request.query_params.get('languages') else []
+        queryset = queryset.filter(language__in=languages) if languages else queryset
```

Comment:
Not introduced by this PR, but I think this could be made slightly more readable:
```suggestion
        language = request.query_params.get("language")
        if language:
            queryset = queryset.filter(language__in=language.split(","))
```